src/tools/rados/rados.cc

```diff
@@ -120,6 +120,10 @@ void usage(ostream& out)
 "   listwatchers <obj-name>          list the watchers of this object\n"
 "   set-alloc-hint <obj-name> <expected-object-size> <expected-write-size>\n"
 "                                    set allocation hint for an object\n"
+"   set-redirect <object A> --target-pool <caspool> <target object A>\n"
+"                                    set redirect target\n"
+"   set-chunk <object A> --target-pool <caspool> <chunk-length>\n"
+"                                    convert an object to chunked object\n"
```

Comment:
...and then the set-chunk should be switched around to map to the above. Like, a list of "offset length target_oid target_offset"?

Reply:
Fixed